[2021-01-15 16:37:02,041] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op2 2021-01-15T07:35:03.242575+00:00 [queued]>
[2021-01-15 16:37:02,048] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op2 2021-01-15T07:35:03.242575+00:00 [queued]>
[2021-01-15 16:37:02,048] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:37:02,048] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-01-15 16:37:02,048] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:37:02,053] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): op2> on 2021-01-15T07:35:03.242575+00:00
[2021-01-15 16:37:02,055] {standard_task_runner.py:54} INFO - Started process 9161 to run task
[2021-01-15 16:37:02,105] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'noti_test', 'op2', '2021-01-15T07:35:03.242575+00:00', '--job_id', '56', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/noti_test.py', '--cfg_path', '/var/folders/4f/xnn880bn51d9945mdydbgwdw0000gn/T/tmp52p0chuy']
[2021-01-15 16:37:02,107] {standard_task_runner.py:78} INFO - Job 56: Subtask op2
[2021-01-15 16:37:02,129] {logging_mixin.py:112} INFO - Running <TaskInstance: noti_test.op2 2021-01-15T07:35:03.242575+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-15 16:37:02,139] {logging_mixin.py:112} INFO - function2
[2021-01-15 16:37:02,140] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-01-15 16:37:02,142] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=noti_test, task_id=op2, execution_date=20210115T073503, start_date=20210115T073702, end_date=20210115T073702
[2021-01-15 16:37:07,047] {local_task_job.py:102} INFO - Task exited with return code 0
